fhir-server-test/src/test/java/com/ibm/fhir/server/test/BundleTest.java

```diff
@@ -2209,8 +2209,7 @@ public void testBatchConditionalDeletes() throws Exception {
         assertResponseBundle(responseBundle, BundleType.BATCH_RESPONSE, 4);
         assertGoodGetResponse(responseBundle.getEntry().get(0), Status.OK.getStatusCode(), HTTPReturnPreference.MINIMAL);
         assertGoodGetResponse(responseBundle.getEntry().get(1), Status.NO_CONTENT.getStatusCode(), HTTPReturnPreference.MINIMAL);
-        assertBadResponse(responseBundle.getEntry().get(2), Status.PRECONDITION_FAILED.getStatusCode(),
-                "returned multiple matches");
+        assertGoodGetResponse(responseBundle.getEntry().get(2), Status.NO_CONTENT.getStatusCode(), HTTPReturnPreference.MINIMAL);
```

Comment:
this is the only real change in this file, all the others are just formats like removing empty spaces at the line ends.